Wiki reorg: Staking Dashboard (#4615)
w3f/polkadot-wiki: docs/general/staking-dashboard.md

For more information about staking on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
visit the [staking page](../learn/learn-staking.md) and the
[advanced staking page](../learn/learn-staking-advanced.md).

:::

The [Polkadot Staking Dashboard](https://staking.polkadot.network/#/overview) is a tool only
dedicated to [staking](../learn/learn-staking.md) on Polkadot,
[Kusama](../learn/learn-kusama-vs-polkadot.md) (Polkadot's canary network) and
[Westend](../maintain/maintain-networks.md#westend-test-network) (Polkadot's test network).

The dashboard is not a wallet, meaning that you cannot transfer funds between accounts. To fund
accounts, you can use [third-party wallets](./wallets.md) or the
[Polkadot-JS UI](./polkadotjs-ui.md). You can participate in staking by being a
[nominator](../learn/learn-nominator.md) or a member of a
[nomination pool](../learn/learn-nomination-pools.md). Once you have a funded account, you need to
connect it to the dashboard (connect button on the top-right corner), this will allow you to
interact with {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} native staking.

:::info The dashboard is a Web3 dApp

> The dashboard is a decentralized application (dApp), meaning that under the hood there are no
> centralized servers but a network of nodes storing information about the Polkadot blockchain and
> validating transactions, including everything you decide to do on the dashboard. To login into the
> dashboard you do not need emails and passwords but just your public address. From now on, you are in
> charge of your data and will need to authorize anything you want to do. Welcome to Web3!

Reviewer: This is true, assuming the user is connected to the light client. The RPC servers connected to may still be run by centralized entities. I'm removing the ambiguity here.

```suggestion
The dashboard is a decentralized application (dApp) and to login, you do not need to sign up with an email and password, but just need an account created on the supported non-custodial wallets. Any transaction to be submitted, needs to be signed by you. Also, if you use the light client which is resistant to censorship, you are interacting trustlessly with the network without any intermediaries. Welcome to the world of true Web3!
```

  a Pool", and "Not Staking". The amount that is not staking is further divided into "Free" (a
  balance that can be transferred) and "Reserve" (a balance that is needed the keep the account
  on-chain, see [existential deposit](../learn/learn-accounts.md#existential-deposit-and-reaping)).
  In this case 0.301 KSM are bonded for nominating, 0.3 KSM are bonded in a pool, and 0.145 KSM are
  not used for staking. Of the non-staking balance, 0.144 KSM are free while
  {{ polkadot: <RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={333000000} filter="humanReadable"/> :polkadot }}
  are reserved for the existential deposit.

- **Section F: The Recent Payouts Panel** shows a bar chart with the rewards paid out to you in the
  past 15 days either as a nominator or a pool member (manually claimed). Note how the 4th of April
  bar has the tip that is not filled with color. This is to show a pending payout (for nominators
  only). Below the bar chart is a line chart showing the 10-day moving average.

Additional statistics can be found at the bottom of the overview page.

:::info Pool funds are in system accounts

> Remember that funds bonded in a pool are transferred to the pool's system account, which is not
> accessible to anybody. Some wallets might not display the balance bonded in pools, but the dashboard
> will always show it.

Reviewer: ```suggestion
Remember that funds bonded in a pool are transferred to the pool's system account, which can only be 
accessed by the protocol, and not by any individual user. Some wallets might not display the balance bonded in pools, but the dashboard
will always show it.
```

  that can be transferred) and "Reserve" (balance that is needed the keep the account on-chain, see
  [existential deposit](../learn/learn-accounts.md#existential-deposit-and-reaping)). In this case
  0.301 KSM are bonded for nominating, 0.3 KSM are bonded in a pool, and 0.145 KSM are not used for
  staking. Of the non-staking balance 0.144 KSM are free while
  {{ polkadot: <RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={333000000} filter="humanReadable"/> :polkadot }}
  are reserved for the existential deposit.

- **Section F: The Recent Payouts Panel** shows a bar chart with the rewards that have been paid out
  to you in the past 15 days either as a nominator or a pool member (manually claimed). Note how the
  bar of the 4th of April has the tip that is jot filled with color. This is to show a pending
  payout (for nominators only). Below the bar chart there is a line chart showing then 10-day moving
  average.

Additional statistics can be found at the bottom of the overview page. Note also the current
exchange rate for the native token {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} at the

> footer of the dashboard. This information is used to estimate how much your balance is worth.

Reviewer 2: The exchange rate is not shown on staking.polkadot.network, only in development and staging on Github. Therefore it may be best to remove this sentence as to not confuse users.

unstake. The dashboard will automatically check this for you and prompt a banner.

:::

- **Section C: The Nominator Panel** shows the current status of the nominator, currently set to
  "Nominating and Earning Rewards", and next to it the `Unstake` button allows you to unstake the
  whole bonded amount and stop nominating. In the middle of the panel there is the payout
  destination currently set to "Compounding" (i.e. rewards are added to the bonded funds). The
  `Update` button will allow you to change the destination to "To Your Account" (to the stash
  account as a free balance), "To Another Account" (an account that is not the stash or controller),
  "To Controller Account", or "None" (no payout destination). The bottom part of the panel shows the
  controller account, currently set to KSM Stash (the stash is also the controller). The `Change`
  button allows you to set a controller.
- **Section D: The Nominations panel** shows your nominations and allows you to stop all nomination
  with the `Stop` button or to select specific validators (`Select` button) and stop nominating only
  those.

> ### Payout

Reviewer 2: ```suggestion
### Payouts
```

This page is an expanded version of Panel F in the [Overview](#overview) page. It also shows all

> validators that paid out rewards to your accounts in the past few months.

Reviewer 2: ```suggestion
validators and / or nomination pools that paid out rewards to your accounts in the past few months.
```
